cluster-autoscaler/simulator/delta_cluster_snapshot.go

```diff
@@ -180,6 +181,7 @@ func (data *internalDeltaSnapshotData) clearCaches() {
 func (data *internalDeltaSnapshotData) clearPodCaches() {
 	data.havePodsWithAffinity = nil
 	data.havePodsWithRequiredAntiAffinity = nil
+	data.pvcNamespaceMap = nil
```

Comment:
Can you add a TODO here to update the cache when adding/removing pods instead of invalidating the whole cache?

Reply:
done.